data/distro.yml

```diff
@@ -564,30 +564,41 @@
       <p>Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
     </ol>
 
-- name: "SulinOS"
-  logo: "sulinos.svg"
+- name: "Turkman Linux"
+  logo: "turkman.svg"
   info: >
     <ol class="distrotut">
       <li>
         <h2>Install Flatpak</h2>
-        <p>To install Flatpak, run the following in a terminal:</p>
+        <p>To install Flatpak on Turkman Linux, run the following in a terminal:</p>
+        <pre><code>
+          <span class="unselectable">Emerge way</span>
+          <span class="unselectable">#</span> ymp install build-base --no-emerge
```

Comment:
Is the `--no-emerge` switch correct here if it is an "emerge way"? :-)

Reply:
build-base 8ackage provide emerge tools and cannot emerge itself. We must install with --no-emerge for enable emerge way